app/app_test.go

```diff
@@ -35,7 +37,7 @@ func (ms *MockSink) Send(spans []*types.Span) error {
 func (ms *MockSink) Start() error { return nil }
 func (ms *MockSink) Stop() error  { return nil }
 
-func TestMissingJSONTimestampHandling(t *testing.T) {
+func TestMissingJSONTimestampHandling_V1(t *testing.T) {
```

Comment:
Any particular reason to use the underscore in the test names instead of plain camel case?

Reply:
That's a naming convention I've seen used elsewhere -- different iterations of the same "base" test using an underscore + identifier.

Your comment actually pushed me to try and find a source for it, and I see it's used _sometimes_ in the standard lib, i.e: https://golang.org/src/sort/sort_test.go, for benchmarks:

```
BenchmarkSortString1K
BenchmarkSortString1K_Slice
```

There's also a `TestSortLarge_Random`. (although it's the only TestSortLarge in the package).

This [article](https://ieftimov.com/post/testing-in-go-naming-conventions/) also mentions it and brings an example from Hashicorp/consul:

```
func TestAPI_AgentServices(t *testing.T) {
	// Snipped...
}

func TestAPI_AgentServicesWithFilter(t *testing.T) {
	// Snipped...
}
```

But apparently it's nothing official. 

If you prefer, we can stick to plan camel case here :)